server/repository/src/mock/item_variant.rs

```diff
@@ -0,0 +1,58 @@
+use crate::{item_variant::item_variant_row::ItemVariantRow, mock::item::*};
+
+pub fn mock_item_a_variant_1() -> ItemVariantRow {
```

Comment:
Decided we'd reached the threshold to deserve mocks.
Haven't refactored any other test cases though. I think that's ok right?

Reply:
Sounds fine to me, if we are back i n those test cases can always refactor later 🙏